src/schemas/v2/combined.json

```diff
@@ -36814,6 +36814,182 @@
+                  "Others"
+                ]
+              },
+              "tenure": {
+                "title": "Ownership Type",
+                "description": "What type of ownership is the property?",
+                "type": "string",
+                "enum": [
+                  "Freehold",
+                  "Managed Freehold",
+                  "Commonhold",
+                  "Leasehold",
+                  "Other"
+                ]
+              },
+              "BasicDetails": {
+                "title": "Basic Property Details",
+                "type": "object",
+                "properties": {
+                  "bedrooms": {
+                    "title": "Number of bedrooms",
+                    "type": "integer"
+                  },
+                  "bathrooms": {
+                    "title": "Number of bathrooms",
+                    "type": "integer"
+                  },
+                  "yearBuilt": {
+                    "title": "Built Year",
+                    "type": "string",
+                    "format": "date"
+                  },
+                  "floorArea": {
+                    "title": "Floor area",
+                    "type": "object",
+                    "properties": {
+                      "inSquareMeter": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure meters",
+                            "type": "number"
+                          },
+                          "pricePerSquareMeter": {
+                            "title": "Price per square meter",
+                            "type": "number"
+                          }
+                        }
+                      },
+                      "inSquareFoot": {
+                        "type": "object",
+                        "properties": {
+                          "area": {
+                            "title": "Area in sqaure foot",
+                            "type": "number"
+                          },
+                          "pricePerSquareFoot": {
+                            "title": "Price per square foot",
+                            "type": "number"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "sourceUrls": {
+                "title": "Urls that point to property images, maps, etc.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "title": "URL",
+                      "type": "string",
+                      "format": "uri"
+                    }
+                  }
+                }
+              },
+              "propertyPricing": {
+                "title": "Property Pricing Estimates",
```

Comment:
should we include details of the methodology used here? Where does the credibility of the estimate come from?

Reply:
I think `dataSources` and `estimateMethodology` are the right names here - the credibility comes from the sources...

Reply:
yes naming of properties like `dataSources` and `estimateMethodology` is reasonable, and they provide clear information about purpose. However, it's important to provide additional clarity in schema to ensure understanding. we can add a new property priceEstimationDetails within the `propertyPricing` object to describe the credibility and methodology in more detail. 
'priceEstimationDetails' encompasses both the credibility sources and the methodology used for property price estimation, making it clear and straightforward.